WorkflowConcurrency/Sources/AsyncOperationWorker.swift

```diff
@@ -60,3 +59,85 @@ public struct AsyncOperationWorker<OutputType>: Worker {
         true
     }
 }
+
+extension RenderContext {
+    /// Creates or updates a child workflow of the given type, performs a render
+    /// pass, and returns the result.
+    ///  
+    /// The receiver is the context with which the workflow will be rendered.
+    ///  
+    /// - Returns: The `Rendering` generated by the workflow.
+    /// `WorkflowType`.
+    /// - Parameters:
+    ///   - keyFile: A string that helps uniquely identifies this workflow.
+    ///   - keyFunction: A string that helps uniquely identifies this workflow.
+    ///   - keyLine: An integer that helps uniquely identifies this workflow.
+    ///   - block: Any async closure or function to run. The return `Action`
+    ///   type must match the context's `WorkflowType`.
+    public func run<Action>(
+        keyFile: StaticString = #file,
+        keyFunction: StaticString = #function,
+        keyLine: Int = #line,
+        block: @escaping () async -> Action
+    ) where Action: WorkflowAction, Action.WorkflowType == WorkflowType
+    {
+        AsyncOperationWorker(block)
```

Comment:
nit: It might be worth implementing this method in terms of `run(key:block:)`

Reply:
+1